nef/Extensions/URL+Extension.swift

```diff
@@ -5,15 +5,15 @@ import BowEffects
 
 extension URL {
     func outputURL(command: Command) -> IO<OpenPanelError, URL> {
-        let filename = "nef-\(command) \(Date.now.human)"
+        let filename = "nef-\(command.menuKey) \(Date.now.human)"
         let url = appendingPathComponent(filename).appendingCommandExtensionComponent(command: command)
         return IO.pure(url)^
```

Comment:
This should not be implemented as `IO.pure`, as it is not pure. Every invocation returns a different thing because `Date.now` evaluates to a different value. It should be something like:

```swift
IO.invoke {
    let filename = "nef-\(command.menuKey) \(Date.now.human)"
    let url = appendingPathComponent(filename).appendingCommandExtensionComponent(command: command)
    return url
}
```

Reply:
Good seen!

Reply:
But it is out of this PR, it will be done in the [next one](https://github.com/bow-swift/nef-plugin/pull/18)